pkg/params/run.go

```diff
@@ -39,6 +39,7 @@ func StringToBool(s string) bool {
 
 // WatchConfigMapChanges watches for provide configmap
 func (r *Run) WatchConfigMapChanges(ctx context.Context, run *Run) error {
```

Comment:
this can be also improved the func is on `*Run`, no need to pass `run *Run` as param

Reply:
can be done in a followup

Reply:
Yes but thought to do as part of other PR because need to change test as well

Reply:
ah wait no test changes needed
will update